sdk/spring/spring-cloud-azure-test-eventhubs-binder/src/test/resources/application.yaml

```diff
@@ -7,6 +7,7 @@ spring:
           checkpoint-store:
             account-name: ${STORAGE_ACCOUNT_TEST_EVENTHUB}
             account-key: ${STORAGE_ACCOUNT_KEY_TEST_EVENTHUB}
+            endpoint: ${PRIMARY_STORAGE_BLOB_ENDPOINT}
```

Comment:
Do we have an environment variable for indicating the current cloud type? Because we want to test a test that mimics the user's behavior when configuring different sovereign clouds. In Spring's approach, we'd like users to specify a cloud type instead of the endpoint.

Reply:
If you want to determine what cloud the code is running in, I think the best way currently is to look at `AZURE_AUTHORITY_HOST` which will be one of `https://login.microsoftonline.com` (public), `https://login.chinacloudapi.cn` (china), or `https://login.microsoftonline.us` (usgov). Those endpoint values may also exist on the credential context?

If you want a variable that the user can set, I don't think there's a high level solution for java yet.

Reply:
I see, thanks @benbp